ynr/apps/candidates/tasks.py

```diff
@@ -0,0 +1,84 @@
+from django.conf import settings
+from django.core.management import call_command
+from django_q.models import Schedule
+from django_q_registry import register_task
+
+
+@register_task(
+    name="Process images in moderation queue",
+    schedule_type=Schedule.CRON,
+    cron="* * * * *",
+)
+def moderation_queue_process_queued_images():
+    if settings.ENABLE_SCHEDULED_JOBS:
+        call_command("moderation_queue_process_queued_images")
+
+
+@register_task(
+    name="Parse raw data from SOPNs",
+    schedule_type=Schedule.CRON,
+    cron="* * * * *",
+)
+def sopn_parsing_process_unparsed():
+    if settings.ENABLE_SCHEDULED_JOBS:
+        call_command("sopn_parsing_process_unparsed")
```

Comment:
In the existing crontab, we are running this command with `flock` to prevent overlapping executions.
Django-Q2 doesn't have any built in features relating to help us prevent this like de-duping. If you schedule a task to run every minute that sometimes takes >1 minute then you will get overlapping runs.
I haven't done it, but one option is to implement a lock at the python layer in the task.
Another thing we could do is ignore it and commit to getting to a point where this is happening in response to an event/trigger rather than on a schedule ahead of the May 2026 elections (I assume this basically doesn't matter right now?)
Not sure what your future thoughts/plans are for this one